src/auto/functions.rs

```diff
@@ -514,6 +514,11 @@ pub fn file_set_contents<P: AsRef<std::path::Path>>(
     }
 }
 
+//#[cfg(any(feature = "v2_66", feature = "dox"))]
+//pub fn file_set_contents_full<P: AsRef<std::path::Path>>(filename: P, contents: &[u8], flags: /*Ignored*/FileSetContentsFlags, mode: i32) -> Result<(), Error> {
```

Comment:
This can easily be generated if you add the new enum to the `generate` list in `Gir.toml`